static/app/utils/analytics/issueAnalyticsEvents.tsx

```diff
@@ -311,6 +311,6 @@ export const issueEventMap: Record<IssueEventKey, string | null> = {
   'issue_details.sourcemap_wizard_copy': 'Issue Details: Sourcemap Wizard Copy',
   'issue_details.sourcemap_wizard_learn_more':
     'Issue Details: Sourcemap Wizard Learn More',
-  'issue_details.open_replay_details_clicked':
-    'Issue Details: Open Replay Details Clicked',
+  // 'issue_details.open_replay_details_clicked':
+  //   'Issue Details: Open Replay Details Clicked',
```

Comment:
@ryan953 Should this be commented out?

Reply:
this actually depends on https://github.com/getsentry/sentry/pull/56894

so i'll have a merge conflict